avm/ptn/ai-platform/baseline/tests/e2e/max/main.test.bicep

```diff
@@ -113,7 +114,7 @@ module testDeployment '../../../main.bicep' = [
       virtualMachineConfiguration: {
         enabled: true
         name: take('${namePrefix}-vm-${serviceShort}', 15)
-        zone: 1
+        zone: 0
```

Comment:
Disabling availability zones, so the VM can be deployed in the default region (West Europe)